app/src/components/utils.js

```diff
@@ -1,6 +1,27 @@
 const { v4: uuidv4 } = require('uuid');
+const log = require('npmlog');
 
 module.exports = {
+
+  /**
+  * @function determineCarboneErrorCode
+  * We want to return 422s if the template has a user error in it's construction.
+  * Carbone doesn't throw specific errors in this case, so we'll do a best-effort of
+  * determining if it should be a 422 or not (keep doing a 500 in any other case)
+  * @param {err} String The thrown exception from Carbone
+  * @returns {integer} The output filename for the response
+  */
```

Comment:
Missing indent of 2 spaces in this block JSDoc

Reply:
👍